math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -40,6 +40,21 @@
 namespace ROOT {
 
 namespace VecOps {
+
+namespace Internal {
+
+template <typename... Args>
+void REmplaceBackHelper(std::vector<bool>& v, Args... args) {
+   v.push_back(std::forward<Args>(args)...);
```

Comment:
here and (especially, because of move semantics) below, needs a `std::forward<Args>(args)...`

Reply:
but there is one? what do i miss?

Reply:
pfff nothing sorry, am tired...